helm/values.yaml

```diff
@@ -176,6 +176,11 @@ namespacedValidation:
 automaticChildApproval:
   enabled: true
 
+# when updating a resource, without changing the image reference (e.g. scaling up a deployment), the
```

Comment:
I think I'd make comment more similar to above and bit more straight-forward like `feature_name/config_name determines how resource updates that do not alter the image are handled during admission. If set to true, no validation will occur and the update will be admitted. If set to false (default), validation is performend, ensuring that the image has an up-to-date signature.